crates/rooch-types/src/crypto.rs

```diff
@@ -109,7 +111,7 @@ impl RoochKeyPair {
     pub fn private(&self) -> &[u8] {
         match self {
             RoochKeyPair::Ed25519(kp) => kp.as_bytes(),
-            RoochKeyPair::Secp256k1(kp) => kp.as_bytes(),
+            RoochKeyPair::Secp256k1(kp) => kp.secret.as_bytes(),
```

Comment:
Why use `.secret` here? Should we also change the `Ed25519` ?

Reply:
> Why use `.secret` here? Should we also change the `Ed25519` ?

The Ed25519's public key can not be publicly accessible, so it has to be remained to `kp.as_bytes()` but we can change the function name to `kp_bytes` instead of `private`.

Reply:
Use `as_ref()` should be more appropriate to get the private keys.